docs/docs/user/known-instances.md

```diff
@@ -31,6 +31,9 @@ This page contains a non-exhaustive list with all websites using Anubis.
 - https://codeberg.org/
 - https://www.cfaarchive.org/
 - https://gitlab.freedesktop.org/
+- https://bugzilla.proxmx.com
```

Comment:
Just in case, you probably meant https://bugzilla.proxmox.com?

Reply:
I did. New rule: no making docs PRs before the caffeine sets in lol.